roles/pulp_content_guard/tasks/rbac/rbac.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: Get RBAC content guard list
+  uri:
+    url: "{{ pulp_rbac_cg_url }}"
+    user: "{{ pulp_username }}"
+    password: "{{ pulp_password }}"
+    method: GET
+    status_code: 200
+    force_basic_auth: true
+  no_log: true
+  register: rbac_cg_list_result
+
+- name: Set fact remove_rbac_cg
+  set_fact:
+    remove_rbac_cg: "{{ (remove_rbac_cg | default([])) + [item.name] }}"
+  when: item.state | default('present') == 'absent'
+  with_items: "{{ pulp_content_guard_rbac }}"
+
+- name: Create RBAC content guards
+  vars:
+    rbaccgnames: "{{ rbac_cg_list_result.json.results | map(attribute='name') | list }}"
+  uri:
+    url: "{{ pulp_rbac_cg_url }}"
+    user: "{{ pulp_username }}"
+    password: "{{ pulp_password }}"
+    force_basic_auth: true
+    method: POST
+    status_code: 201
+    body:
+      name: "{{ item.name }}"
+      description: "{{ item.description | default(omit) }}"
+    body_format: form-urlencoded
+  loop: "{{ pulp_content_guard_rbac | default([], true) }}"
+  loop_control:
+    label: "{{ item.name }}"
+  # no_log: true
```

Comment:
Required or not?

Reply:
If you're referring to the no_log parameter, then I think it probably is as the play handles passwords. There's a few places that I've commented them out just for debugging. I'll do a pass later on and add them all back in. 

If you're referring to the loop control label, I was experimenting with adding it to make the plays clearer as they run, however I recently removed it as I'm not sure helped much because item doesn't contain a huge amount of data

Reply:
Usually comments refer to the line above.

The uri Ansible module should handle censoring the password argument. It's best to use no_log sparingly as it makes debugging difficult